app/scripts/modules/ecs/pipeline/stages/enableAsg/ecsEnableAsgStage.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const angular = require('angular');
+
+import { PipelineTemplates, StageConstants } from '@spinnaker/core';
```

Comment:
Don't need to import `PipelineTemplates` anymore.

Reply:
Done